docs/corectl_config.md

```diff
@@ -0,0 +1,105 @@
+## corectl config
+
+With `corectl` it is possible to configure values that should be passed to your command. This can be configured in a `yaml` file residing in e.g. an repo or locally on your computer.
+By default `corectl` will pick up a `corectl.yml | corectl.yaml` file from your current directory. It is also possible to pass a specific configuration file using the `--config` or `-c` flag.
+
+All properties set in a configuration file can be overriden by passing another value as a flag instead.
+
+### Configuration properties
+
+Below is a configuration example utilizing the different properties that are available today:
+
+```yaml
+engine: localhost:9076 
+app: project1.qvf
+script: ./dummy-script.qvs
+connections:
+  myconnection:
+    type: testconnector
+    username: gwe
+    settings:
+      host: corectl-test-connector
+  myfolderconnection:
+    connectionstring: /data
+    type: folder
+objects:
+  - ./object-*.json
+measures:
+  - ./*measure*.json
+dimensions:
+  - ./dimension-*.json
+headers:
+  authorization: "Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiJmb2xrZSJ9.MD_revuZ8lCEa6bb-qtfYaHdxBiRMUkuH86c4kd1yC0" #generated at jwt.io with the password passw0rd
+```
+
+### engine
+
+This property sets the URL to the engine instance that you want `corectl` to connect to by default. Can be overriden with the `-e` or `--engine` flag.
+
+```yaml
+engine: localhost:9076 
+```
+
+### app
+
+This property describes the default app name that should be used for the commands. Typically used when working against a specific app on your engine instance. If you want to open a different app than configured in the file, you can also pass the `--app` or `-a` flag.
```

Comment:
app names without .qvf works as well. Not sure if this is a feature and should be mentioned.

Reply:
I think we should only document the preferred way, which is including the extension in the name imo.